cortado-core/cortado-mockclasses/src/main/java/edu/utexas/cs/utopia/cortado/mockclasses/WaitUntil.java

```diff
@@ -0,0 +1,9 @@
+package edu.utexas.cs.utopia.cortado.mockclasses;
+
+// TODO: remove this
```

Comment:
Can we remove this class?

I assume we are not porting legacy benchmark implementations